build/config/tv/apps-engineering.list

```diff
@@ -22,7 +22,6 @@ apps/gallery
 apps/keyboard
 apps/music
 apps/pdfjs
-apps/system
```

Comment:
I guess this deletion raises a GBI test fail. We may need to fix it or just leave apps/system and resolve it in another bug.